hedera-node/hedera-smart-contract-service-impl/build.gradle.kts

```
@@ -33,8 +33,9 @@ dependencies {
     implementation(project(":hedera-node:hedera-mono-service"))
 
     testImplementation(testLibs.bundles.testing)
-    testImplementation(testFixtures(project(":hedera-node:hedera-mono-service")))
     testImplementation(testLibs.mockito.inline)
     testImplementation(project(mapOf("path" to ":hedera-node:hedera-app-spi")))
     testImplementation(testFixtures(project(":hedera-node:hedera-app-spi")))
+    testImplementation(project(mapOf("path" to ":hedera-node:hedera-mono-service")))
```

What benefit does this syntax (mapOf) give to us?

---

I am not sure. Followed similar pattern of ` testImplementation(project(mapOf("path" to ":hedera-node:hedera-app-spi")))`. Is that not needed ?

---

No idea who did that :D I would assume the following just works:
```suggestion
    testImplementation(project(":hedera-node:hedera-mono-service"))
```

---

But this should not block the PR. I can have a look later if it is not working :)

---

@hendrikebbers Thank you ! Changed all the places where there is `mapOf`